dotcom-rendering/playwright/tests/parallel-5/atom.video.e2e.spec.ts

```diff
@@ -297,7 +298,7 @@ test.describe('YouTube Atom', () => {
 		// Listen for the ophan call made when the video is played
 		const ophanPlayEventPromise2 = interceptOphanPlayEvent({
 			page,
-			id: 'gu-video-youtube-1e89d5bd-489e-470a-857e-4f30e85b5aec',
```

Comment:
Did we expect this to be the same id as the first video?

Reply:
I think so. Aiui Ophan doesn't care about duplicate videos it just wants to log each video play.

Actually if this is the elementId it looks like a bug as I would have expected we give Ophan the `videoId` (`assetId` prop in `YoutubeBlockComponent.importable`) and not the elementId which can change across renders.

https://github.com/guardian/dotcom-rendering/blob/825743247e243269fff21b8f6a593be6ce117649/dotcom-rendering/src/components/YoutubeBlockComponent.importable.tsx#L186

Worth checking with the Ophan team what their expectations are for these events.

Reply:
So correctly done in `renderElement` but not `Card`. Quite confusing—I'll update.